src/components/legend/draw.js

```diff
@@ -408,41 +410,98 @@ function setupTraceToggle(g, gd) {
         .attr('pointer-events', 'all')
         .call(Color.fill, 'rgba(0,0,0,0)');
 
-    traceToggle.on('click', function() {
+
+    traceToggle.on('mousedown', function() {
```

Comment:
I wonder if we could reuse the [dragelement](https://github.com/plotly/plotly.js/blob/master/src/components/dragelement/index.js) abstraction here? This is a non- :no_entry: comment of course.